docs/cloud-native-security/session-view.asciidoc

```diff
@@ -92,6 +92,11 @@ Session View includes additional badges not pictured above:
 //[role="screenshot"]
 //image::images/session-view-script-button.png[The Script button]
 
+* The alert badge for multiple alerts. If a single event causes alerts of multiple types, multiple icons appear on the alert badge (a gear, a page, and a globe for `process`, `file`, and `network` alerts, respectively):
```

Comment:
I hate to suggest the dreaded _table_ in Asciidoc, but this seems like a perfect opportunity for a small table to identify each icon more visually, or even just a bulleted list with in-line icons. Juggling this information with a "respectively," and mentally converting from text description to image, lining that up in sequence with the list of respective alert types — it doesn't require a ton of mental effort, but still feels like unnecessary effort to put onto the reader. 

A table or list with each icon illustrated is a lot easier to understand at a glance:

Icon | Alert type
-- | --
<img width="20" alt="image" src="https://user-images.githubusercontent.com/87339667/211056419-edcb0b6f-8fb6-4126-b699-5fc74e76421c.png">  | `process`
<img width="20" alt="image" src="https://user-images.githubusercontent.com/87339667/211056523-98d88795-be92-419d-9a1e-63df4ab2ab03.png">  | `file`
<img width="20" alt="image" src="https://user-images.githubusercontent.com/87339667/211056580-dd95fa9f-8d97-4219-96fc-310fc6634689.png"> | `network`